hellosign_sdk/hsclient.py

```diff
@@ -1939,11 +1942,12 @@ def _send_signature_request(self, test_mode=False, client_id=None, files=None,
             "subject": subject,
             "message": message,
             "signing_redirect_url": signing_redirect_url,
-            "form_fields_per_document": form_fields_per_document,
+            "form_fields_per_document": HSFormat.format_json_data(form_fields_per_document),
```

Comment:
This change assumes that we've changed the expected type for `form_fields_per_document` from `str` to `Dict`. I think it makes sense that we'd expect Dict there, but we can probably make that change in the `python3` branch along with other breaking changes.

Can you remove it from this PR for now (including the definition of `format_json_data` in `hsformat.py`

Reply:
I haven't changed the input format of the data, I have made the change inline with that the required input format is as listed in the docs

The function I have defined is also being used by another parameter so is still useful in this regard.

Reply:
Hey @neillom, here's an example of what the breaking change is:
If you use this [test](https://gist.github.com/martinytodorov/29420b634a173a107db0dfdbf3ef93f9) signature request on your sdk **before** adding the `"form_fields_per_document": HSFormat.format_json_data(form_fields_per_document),` change, you will get a successful response (200).

If you try to make the very same request **after** your change is in place, you will get:
```
{
        "error": {
                "error_msg": "1 signer has not been assigned to fields in form_fields_per_document",
                "error_name": "bad_request"
        }
}
```

Reply:
I see the difference in the requests now. During my testing I read the documentation and followed the examples (copy/paste) by declaring it not as a string as I had followed the general trend of other fields (custom_fields, signers etc). Is there an argument for this rather than passing a string? The open issue is for passing a JSON object.

Reply:
@alexmac05 how should we handle this case? I can see we have active customers using `form_fields_per_document` with the current version of the python-sdk?

I agree with @neillom that this makes more sense to be a Dict than str, but I don't want to introduce a breaking change now since we want to focus our efforts into python3?

Reply:
What if I included logic to format the string to a JSON object which would allow both formats to be accepted ?

Reply:
Hey @neillom, I ran this Idea with Api Support and they like it. Let's make it to accept both types. In python 3 we can make it Dict only.